python/ray/worker.py

```diff
@@ -2313,13 +2326,17 @@ def put(value, worker=global_worker):
         return object_id
 
 
-def wait(object_ids, num_returns=1, timeout=None, worker=global_worker):
+def wait(object_ids,
+         num_returns=1,
+         timeout_seconds=None,
+         timeout=None,
```

Comment:
Hopefully that won't happen much, since using positional arguments for keyword arguments is a bad practice.

That said, I'm not sure I understand your suggestion as Python functions can't be overloaded.

Reply:
sorry, I meant something like `ray.future.wait(object_ids, num_returns, timeout_seconds, global_worker)`.

Reply:
I see, thanks. I prefer to update the `ray.wait` API right away.